addons/remix-tree-button/main.js

```diff
@@ -0,0 +1,44 @@
+export default async function ({ addon, global, console }) {
+  //define remix tree button elements
+
+  const subactions = document.querySelector(".flex-row.subactions").querySelector(".action-buttons");
+
+  const remixtree = document.createElement("button");
+  remixtree.className = "button action-button remixtree-button";
+  remixtree.id = "scratchAddonsRemixTreeBtn";
+  remixtree.innerHTML = `
```

Comment:
Don't use innerHTML

Reply:
well it's much more efficient than creating separate image and text elements as variables

Reply:
fixed